config/airflow_local_settings.py

```diff
@@ -1,11 +1,47 @@
+from copy import deepcopy
+
+from kubernetes.client import models as k8s
+from kubernetes.client.models import V1Container, V1Pod
+
 STATE_COLORS = {
-    "queued": 'gray',
-    "running": 'lime',
-    "success": '#0000FF',
-    "failed": 'red',
-    "up_for_retry": 'gold',
-    "up_for_reschedule": 'turquoise',
-    "upstream_failed": 'orange',
-    "skipped": 'pink',
-    "scheduled": 'tan',
+    "queued": "gray",
+    "running": "lime",
+    "success": "#0000FF",
+    "failed": "red",
+    "up_for_retry": "gold",
+    "up_for_reschedule": "turquoise",
+    "upstream_failed": "orange",
+    "skipped": "pink",
+    "scheduled": "tan",
 }
+
+
+def pod_mutation_hook(pod: V1Pod):
+    """https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/V1Pod.md"""
+
+    # check that we're running a prio-processor job, and spin up a side-car
+    # container for proxying gcs buckets.
+    if pod.metadata.labels["job-kind"] == "prio-processor":
+        pod.spec.share_process_namespace = True
+        # there is only one container within the pod, so lets append a few more
+
+        # for proxying gcs resource consistently across platforms
+        minio_container = deepcopy(pod.spec.containers[0])
+        minio_container.image = "minio/minio:RELEASE.2021-06-17T00-10-46Z"
```

Comment:
Probably, but the minio container is pinned because nince newer versions break behavior with the default port on the docker-compose version of the integration tests.